src/btree/bt_vrfy.c

```diff
@@ -662,6 +663,10 @@ __verify_tree(
              * If configured, continue traversing through the pages of the tree even after
              * encountering errors reading in the page.
              */
+            if (ret != 0 && __wt_ref_addr_copy(session, ref, &addr)) {
+                WT_RET(__wt_msg(session, "ERROR unreadable block at: %s",
+                  __wt_addr_string(session, addr.addr, addr.size, vs->tmp2)));
```

Comment:
`"%s Read failure while accessing a page from the column internal page (ret = ""%d).", __verify_addr_string(session, child_ref, vs->tmp1), ret));`

Reply:
noted, fixed in https://github.com/wiredtiger/wiredtiger/pull/10095/commits/3bd0af803179ff0d1eb0ebbdc0edb13fdac34624

Reply:
thank you, fixed https://github.com/wiredtiger/wiredtiger/pull/10095/commits/3bd0af803179ff0d1eb0ebbdc0edb13fdac34624